posthog/tasks/process_event.py

```diff
@@ -1,32 +1,64 @@
 from celery import shared_task
-from posthog.models import Person, Element, Event, Team
+from posthog.models import Person, Element, Event, Team, PersonDistinctId
 from typing import Union, Dict
 from dateutil.relativedelta import relativedelta
 from dateutil import parser
 
 from django.db import IntegrityError
 import datetime
 
-def _alias(distinct_id: str, new_distinct_id: str, team_id: int) -> None:
+def _alias(previous_distinct_id: str, distinct_id: str, team_id: int, retry_if_failed:bool = True) -> None:
     try:
-        person = Person.objects.get(team_id=team_id, persondistinctid__distinct_id=distinct_id)
+        old_person = Person.objects.get(team_id=team_id, persondistinctid__distinct_id=previous_distinct_id)
```

Comment:
I think if you define `old_person: Optional[Person]` you don't have to do `# type: ignore` below

Reply:
Good tip! This is fixed and I also added all the types to make mypy stop complaining about the test.

Reply:
Yep I made mypy stricter for the tasks directory as it's new, so we can keep it to a higher standard :)